packages/devextreme/js/ui/context_menu/ui.context_menu.js

```diff
@@ -556,11 +555,20 @@ class ContextMenu extends MenuBase {
         if(!arg.cancel) {
             this._hideAllShownSubmenus();
             this._setOptionWithoutOptionChange('visible', false);
+
+            const $firstMenuItem = this._overlay.$content().find(`.${DX_MENU_ITEM_CLASS}`).first();
+            const $scrollableElement = $firstMenuItem.closest(`.${SCROLLABLE_CLASS}`);
+            const scrollableInstance = $scrollableElement.dxScrollable('instance');
+
+            scrollableInstance?.scrollTo(0);
```

Comment:
Why we should check ?.scrollTo? I thought that we always render scrollable for contextMenu and submenus.